### ISSUE TITLE
`nitrogql generate` throws error on Vercel's build environment: `bad option: --import=@nitrogql/esbuild-register`

### ISSUE THREAD

**Issue by cm-ayf** [uhyo/nitrogql#24]
Reproduction: https://github.com/cm-ayf/nitrogql-nextjs ([examples/nextjs](https://github.com/uhyo/nitrogql/tree/master/examples/nextjs) with `vercel.json`)

<details>
<summary>Full error log from Vercel:</summary>

```
/node18/bin/node: bad option: --import=@nitrogql/esbuild-register
Error: Command failed: /node18/bin/node --no-warnings --import=@nitrogql/esbuild-register --experimental-loader=@nitrogql/esbuild-register/hook --input-type=module
/node18/bin/node: bad option: --import=@nitrogql/esbuild-register
    at checkExecSyncError (node:child_process:885:11)
    at execFileSync (node:child_process:921:15)
    at executeNodeSync (file:///vercel/path0/node_modules/@nitrogql/core/dist/config.js:27:12)
    at execute_node (file:///vercel/path0/node_modules/@nitrogql/core/dist/config.js:62:24)
    at wasm://wasm/00824b7a:wasm-function[741]:0x1a732f
    at wasm://wasm/00824b7a:wasm-function[21]:0x7925
    at wasm://wasm/00824b7a:wasm-function[1679]:0x1da3bc
    at wasm://wasm/00824b7a:wasm-function[690]:0x1a0f6c
    at wasm://wasm/00824b7a:wasm-function[1718]:0x1dac7c {
  status: 9,
  signal: null,
  output: [
    null,
    '',
    '/node18/bin/node: bad option: --import=@nitrogql/esbuild-register\n'
  ],
  pid: 230,
  stdout: '',
  stderr: '/node18/bin/node: bad option: --import=@nitrogql/esbuild-register\n'
}
Error:
Failed to execute config file: Failed to execute config file
Error: Command "npm run generate && npm run build" exited with 1
```

</details>

This issue does not reproduce with (local) Node.js of either `v18.18.2` or `v20.8.1`, but does reproduce only in Vercel build environment which has been confirmed (just added `node -v` in `buildCommand`) to be using `v18.17.1` currently.

This issue reproduces in `@nitrogql/cli@1.3.2` but does not in `@nitrogql/cli@1.3.0`. (I ended up avoiding this issue by downgrading `@nitrogql/cli` to `v1.3.0` and using `scalar.mjs`; refer to #19.)

[This diff](https://github.com/uhyo/nitrogql/pull/22/files#diff-043e32aa7e10aef06c2a1b98a75bcc52dbe8716f87266ace0a79a949e8083ea0R41-R48) in #22 seems to throw the error. [`node --import`](https://nodejs.org/api/cli.html#--importmodule) is introduced in `v19.0.0` and `v18.18.0`, but Node.js version in Vercel build environment is not sufficient to use this flag.

**Comment by cm-ayf** [comment]
It may be not very good to depend on Node.js feature that is "[Stability: 1](https://nodejs.org/api/documentation.html#stability-index) - Experimental" :thinking:

**Comment by uhyo** [comment]
Thanks you for detailed report 🙂 
Fixed in v1.3.3.

As you suggested, the `--import` flag seems too new for Node.js 18 users. I updated to use the `--require` flag instead.

I'm not very hesitant to using experimental features though, as long as I can actively maintain the project, since experimental features will be experimental forever if no one uses them. 🙃

**Comment by cm-ayf** [comment]
I've confirmed that `@nitrogql/cli@1.3.3` works perfectly on Vercel build environment. Thank you!

By the way, Vercel has just announced that [Node.js v20 LTS is available in beta](https://vercel.com/changelog/nodejs-20).
